internal/infrastructure/kubernetes/ratelimit_infra.go

```diff
@@ -52,10 +52,6 @@ func (i *Infra) CreateOrUpdateRateLimitInfra(ctx context.Context) error {
 
 // DeleteRateLimitInfra removes the managed kube infra, if it doesn't exist.
 func (i *Infra) DeleteRateLimitInfra(ctx context.Context) error {
-	if err := ratelimit.Validate(ctx, i.Client.Client, i.EnvoyGateway, i.Namespace); err != nil {
```

Comment:
curious why this was needed

Reply:
will be fixed in https://github.com/envoyproxy/gateway/pull/4430